src/features/nft/nftModals/changeStatusModal.tsx

```diff
@@ -0,0 +1,85 @@
+import React from 'react'
+// Components
+import { Modal } from '../../../common/components/Modal'
+import { Button } from '../../../common/components/Buttons'
+import Select, { TOption } from '../../../common/components/Select/Select'
+import { Input } from '../../../common/components/Inputs'
+import DatePicker from '../../../common/components/DatePicker'
+import Link from '../../../common/components/Link'
+
+export type TChangeStatusModal = {
+  title?: string
+  isOpen: boolean
+  statusOptions: Array<TOption>
+  listedOptions: Array<TOption>
+  durationOptions: Array<TOption>
+  handleClose?: (event: React.MouseEvent<HTMLButtonElement>) => void
+}
+
+const ChangeStatusModal = ({
+  isOpen,
+  handleClose,
+  title,
+  statusOptions,
+  listedOptions,
+  durationOptions,
+}: TChangeStatusModal): JSX.Element => {
+  const [status, setStatus] = React.useState<TOption | null>(statusOptions[0])
```

Comment:
Can we use `useState<TOption | null>` instead of `React.useState<TOption | null>`? Please also update for below sections